children/README.md

```diff
@@ -0,0 +1,156 @@
+# STAC API - Children
+
+- [STAC API - Children](#stac-api---children)
+  - [Link Relations](#link-relations)
+  - [Endpoints](#endpoints)
+  - [Pagination](#pagination)
+  - [Example](#example)
+
+- **OpenAPI specification:** [openapi.yaml](openapi.yaml) ([rendered version](https://api.stacspec.org/v1.0.0-beta.5/children))
+- **Conformance URI:** <https://api.stacspec.org/v1.0.0-beta.5/children>
+- **Extension [Maturity Classification](../extensions.md#extension-maturity):** Pilot
+- **Dependencies**: [STAC API - Core](../core)
+
+A STAC API can return information about all STAC [Catalogs](../stac-spec/catalog-spec/catalog-spec.md) available using a link
+from the landing page that uses the link relation `children`, which links to an endpoint called
+`/children`. The purpose of this endpoint is to present a single resource from which clients can retrieve
+all the child objects (Catalogs and Collections) of a Catalog. This eliminates then need for a client to
```

Comment:
I think that returning both Catalogs and Collections in the `/children` endpoint is confusing.  My understanding after reading this PR is that collections are for searching while catalogs are for browsing.  It says in the `Browseable Catalogs` best practices that collections should not be included as part of the browseable tree of catalogs.  Allowing collections to be used for browsing (even though its not recommended) is poor separation of concerns.

Reply:
Yeah, this is a bit confusing. Like do I not need to retrieve /collections anymore if /children is implemented? How should clients handle all the duplication to show a list of unique catalogs/collections?

Reply:
I can see the benefit of having `children` return both Catalogs and Collections - it more fits with the STAC core specification which uses `rel=child` for both of those cases. It also allows for all of the catalogs and collections to be retrieved in a single paginated call - useful for rendering the content of the API.

One addition we could make is that a query parameter can determine which type to return - e.g.

```
https://stac.api/children?type=catalog
```

Reply:
It's included as part of https://github.com/philvarner/stac-api-spec/blob/catalogs-and-browseable/core/README.md#browseable-catalogs introduced in this PR.  The exact language is:

> These are the two standard ways of structuring a browseable tree of catalogs, the only difference being whether the Collection is used as part of the tree or not:
> - Catalog (root) -> Catalog* -> Item (recommended)
> - Catalog (root) -> Collection -> Catalog* -> Item

Reply:
I'll see if I explicitly stated it anywhere, but the intention was that `/children` would return the same list of entities that are linked to via `rel=child` from the root. The benefit (as Rob has articulated before) is that a client can get all of the entities with one call instead of having to make one HTTP request for each one to, say, only get the title of the collection or catalog.